src/workflows/filefeed-workflow/filefeed-listener.ts

```diff
@@ -21,13 +21,26 @@ const demo = Client.create((client) => {
     });
   });
 
+  client.on(
+    '**', //listens for any event
+    (event: FlatfileEvent) => {
+      console.log('any event: ' + JSON.stringify(event));
+      const { spaceId } = event.context;
+      const topic = event.topic;
+
+      post({
+        hostname: 'hcm.show',
+        // hostname: 'bad2-108-27-27-221.ngrok-free.app',
```

Comment:
Can you cut the ngrok comment and all the commented out stuff too? We've been collecting a lot of not used code here haha